web/src/recent_view_ui.ts

```diff
@@ -1133,8 +1140,13 @@ function recenter_focus_if_off_screen(): void {
         const topic_center_y = (position.top + position.bottom) / 2;
 
         const topic_element = document.elementFromPoint(topic_center_x, topic_center_y);
-        assert(topic_element !== null);
-        row_focus = $topic_rows.index($(topic_element).closest("tr")[0]);
+        if (topic_element === null) {
+            // Last visible element is above the vertical center of the table.
+            row_focus = $topic_row.length - 1;
```

Comment:
Is this meant to be `$topic_rows.length`?

Reply:
Sorry, thanks for catching that!